app/controllers/curation_concerns/monographs_controller.rb

```diff
@@ -1,5 +1,10 @@
 class CurationConcerns::MonographsController < ApplicationController
   include CurationConcerns::CurationConcernController
   set_curation_concern_type Monograph
+  # Use the Monograph Presenter
+  def show_presenter
```

Comment:
This is already set on line 9 (new way), so there's no need to include these lines.